### ISSUE TITLE
ESM Module error using twind, error came when start create project currently

### ISSUE THREAD

**Issue by darwinprayoga** [denoland/fresh#1107]
✘ [ERROR] [unreachable] Not an ESM module. [plugin deno]

An error occurred during route handling or page rendering. Error: Build failed with 1 error:
error: [unreachable] Not an ESM module.
    at failureErrorWithLog (https://deno.land/x/esbuild@v0.17.11/mod.js:1612:15)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1024:25
    at runOnEndCallbacks (https://deno.land/x/esbuild@v0.17.11/mod.js:1447:45)
    at buildResponseToResult (https://deno.land/x/esbuild@v0.17.11/mod.js:1022:7)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1051:16
    at responseCallbacks.<computed> (https://deno.land/x/esbuild@v0.17.11/mod.js:673:9)
    at handleIncomingPacket (https://deno.land/x/esbuild@v0.17.11/mod.js:728:9)
    at readFromStdout (https://deno.land/x/esbuild@v0.17.11/mod.js:649:7)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1898:11
An error occurred during route handling or page rendering. Error: Build failed with 1 error:
error: [unreachable] Not an ESM module.
    at failureErrorWithLog (https://deno.land/x/esbuild@v0.17.11/mod.js:1612:15)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1024:25
    at runOnEndCallbacks (https://deno.land/x/esbuild@v0.17.11/mod.js:1447:45)
    at buildResponseToResult (https://deno.land/x/esbuild@v0.17.11/mod.js:1022:7)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1051:16
    at responseCallbacks.<computed> (https://deno.land/x/esbuild@v0.17.11/mod.js:673:9)
    at handleIncomingPacket (https://deno.land/x/esbuild@v0.17.11/mod.js:728:9)
    at readFromStdout (https://deno.land/x/esbuild@v0.17.11/mod.js:649:7)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1898:11
An error occurred during route handling or page rendering. Error: Build failed with 1 error:
error: [unreachable] Not an ESM module.
    at failureErrorWithLog (https://deno.land/x/esbuild@v0.17.11/mod.js:1612:15)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1024:25
    at runOnEndCallbacks (https://deno.land/x/esbuild@v0.17.11/mod.js:1447:45)
    at buildResponseToResult (https://deno.land/x/esbuild@v0.17.11/mod.js:1022:7)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1051:16
    at responseCallbacks.<computed> (https://deno.land/x/esbuild@v0.17.11/mod.js:673:9)
    at handleIncomingPacket (https://deno.land/x/esbuild@v0.17.11/mod.js:728:9)
    at readFromStdout (https://deno.land/x/esbuild@v0.17.11/mod.js:649:7)
    at https://deno.land/x/esbuild@v0.17.11/mod.js:1898:11

**Comment by dangdennis** [comment]
Twind hasn't changed in some time. It's a possible Deno regression. 

Workaround script for new projects: 

```
// main.ts
await start(manifest, /* { plugins: [twindPlugin(twindConfig)] } */);
```

In your routes or layout component:
```
<Head>
  <script src="https://cdn.twind.style" crossOrigin={true}></script>
</Head>
```

**Comment by flmng0** [comment]
FYI. This is a duplicate of another issue. See #1103 
Unrelated to twind by the looks of it.

**Comment by darwinprayoga** [comment]
> FYI. This is a duplicate of another issue. See #1103 Unrelated to twind by the looks of it.

yes sir you're right, I found this issue after including twind though but its okay without.. the problem is the deno version

**Comment by hashrock** [comment]
Closed with Deno version 1.32.4 update. Please let me know if it happens again.
